### PR TITLE
preliminary implementation of the item of interest feed

### PR DESCRIPTION
**Pull request by xaanimus**

### CODE REVIEW

File: CampusTour/TableCells/ItemOfInterestTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        
+        let dateLabel = UILabel()
+        dateLabel.text = model.date.description //TODO format better
+        header.addArrangedSubview(dateLabel)
+        
+        return header
+    }
+    
+    func setCellModel(model: Model) {
+        rootStackView?.removeFromSuperview()
```
Comment (jhseung) [inline CampusTour/TableCells/ItemOfInterestTableViewCell.swift]: are you doing this instead of doing prepareforreuse?

**Comment by jhseung** [comment]
Oh btw you can merge this in I finished reviewing it when I left the comment 💃

File: CampusTour/TableCells/ItemOfInterestTableViewCell.swift (hunk 1 above)
Comment (xaanimus) [reply CampusTour/TableCells/ItemOfInterestTableViewCell.swift]: when prepareforreuse gets called I can't give the cell a model

**Comment by xaanimus** [comment]
This PR was included in another merged PR

**Pull request closed without merging**